mwibutsa/rich-form
# Ft create rich form

#### Description of changes made

> This PR adds a form that supports markdown 

#### Type of change


#### How Has This Been Tested?


#### How should this be manually tested?

- After cloning the [repo](https://github.com/mwibutsa/rich-form.git)
- Check out this branch by `ft-create-rich-form`
- Run `yarn or npm install to install the project dependencies
- Run `yarn start` to start the app,  you will see an interactive form on the home page

#### Checklist


#### GitHub issue your changes fix

#### Screenshots

#### EDIT MODE
![image](https://user-images.githubusercontent.com/36665761/111904140-82906180-8a02-11eb-9091-c6837815824a.png)

#### PREVIEW MODE

![image](https://user-images.githubusercontent.com/36665761/111904108-5ecd1b80-8a02-11eb-840d-5640f7298fd7.png)
